baus/datasources.py

```diff
@@ -325,45 +200,40 @@ def costar(store, parcels):
 
 @orca.table(cache=True)
 def zoning_lookup():
-    file = os.path.join(misc.data_dir(),
-                       "2020_11_05_zoning_lookup_hybrid_pba50.csv")
+    
+    file = os.path.join(orca.get_injectable("inputs_dir"), "2020_11_05_zoning_lookup_hybrid_pba50.csv")
     print('Version of zoning_lookup: {}'.format(file))
-    return pd.read_csv(file,
-                       dtype={'id': np.int64},
-                       index_col='id')
+    
+    return pd.read_csv(file, dtype={'id': np.int64}, index_col='id')
 
 
-# zoning for use in the "baseline" scenario
 @orca.table(cache=True)
-def zoning_baseline(parcels, zoning_lookup, settings):
-    file = os.path.join(misc.data_dir(),
-                        "2020_11_05_zoning_parcels_hybrid_pba50.csv")
-    print('Version of zoning_parcels: {}'.format(file))                    
-    df = pd.read_csv(file,
-                     dtype={'geom_id':   np.int64,
-                            'PARCEL_ID': np.int64,
-                            'zoning_id': np.int64},
-                     index_col="geom_id")
-    df = pd.merge(df, zoning_lookup.to_frame(),
-                  left_on="zoning_id", right_index=True)
+def zoning_existing(parcels, zoning_lookup, settings):
+
+    file = os.path.join(orca.get_injectable("inputs_dir"), "2020_11_05_zoning_parcels_hybrid_pba50.csv")
+    print('Version of zoning_parcels: {}'.format(file))
+
+    df = pd.read_csv(file, dtype={'geom_id':   np.int64, 'PARCEL_ID': np.int64, 'zoning_id': np.int64}, index_col="geom_id")
```

Comment:
Oh yeah. I'm hoping the key identifiers will become much more obvious because there will only be one per table (not three variations)! But yaml is great.

Reply:
Noted in: https://app.asana.com/0/385259290425521/1203767459263968